terraform/modules/hub/variables.tf

```diff
@@ -14,6 +14,11 @@ variable "number_of_apps" {
   default = 2
 }
 
+variable "number_of_frontend_apps" {
+  type    = number
+  default = 2
+}
```

Comment:
I wouldn't want to set number of frontend_apps to 2 in production by default.

Reply:
https://github.com/alphagov/verify-infrastructure-config/pull/354 will set it to 16. I'm happy for the default to be 2.